corehq/apps/tour/templates/tour/config/new_blank_app.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
+<script type="text/javascript">
```

Comment:
not really sure there's any better solution but this seems like a lot of scaffolding for each time we set this up. curious if  @dimagi/js-team has any clever ideas about simplifying?